examples/secure_storage/ta/src/main.rs

```diff
@@ -74,15 +74,19 @@ pub fn delete_object(params: &mut Parameters) -> Result<()> {
 pub fn create_raw_object(params: &mut Parameters) -> Result<()> {
     let mut p0 = unsafe { params.0.as_memref().unwrap() };
     let mut p1 = unsafe { params.1.as_memref().unwrap() };
+
     let mut obj_id = vec![0; p0.buffer().len() as usize];
     obj_id.clone_from_slice(p0.buffer());
+    let mut data_buffer = vec![0; p1.buffer().len() as usize];
+    data_buffer.clone_from_slice(p1.buffer());
```

Comment:
@mssun In original code obj_id uses "clone_from_slice".So is it necessary to use "clone_from_slice", or it could be "copy_from_slice" too?

Reply:
Yes.

Reply:
OK.I'll update this.